nixos/modules/services/backup/borgbackup.nix

```
@@ -91,13 +91,17 @@ let
           ++ optional (isLocalPath cfg.repo) cfg.repo;
         PrivateTmp = cfg.privateTmp;
       };
-      environment = {
-        BORG_REPO = cfg.repo;
+      environment = mkBorgEnvironment cfg // {
         inherit (cfg) extraArgs extraInitArgs extraCreateArgs extraPruneArgs;
-      } // (mkPassEnv cfg) // cfg.environment;
+      };
       inherit (cfg) startAt;
     };
 
+  mkBorgEnvironment = cfg: {
```

```suggestion
  mkEnvironment = cfg: {
```
But actually, I don't know why I chose to let those functions have an argument `cfg`. That can just be dropped.

---

OK fixed.

`cfg` is not actually the service module config, but the per-repo config. So it needs to be a variable one way or another.